core/templates/dev/head/domain/dashboard/ShowExplorationTitleSection.js

```diff
@@ -0,0 +1,18 @@
+oppia.directive('explorationTitleSection', function($compile) {
+  return {
+    restrict: 'E',
+    scope: {
+      getExploration: '&exploration',
+      activeExplorationId: '='
```

Comment:
I tried something like this:
```
oppia.directive('explorationTitleSection', function($compile) {
  return {
    restrict: 'E',
    scope: {
      getExploration: '&exploration',
      getActiveExplorationId: '&activeExplorationId'
    },
    link: function(scope, element) {
      scope.exploration = scope.getExploration();
      scope.activeExplorationId = scope.getActiveExplorationId();
```
but this gives some problems. The arrow on clicking a published exploration does not appear.
![image](https://cloud.githubusercontent.com/assets/13691317/19507960/c32ad9e0-95c5-11e6-8671-38acda837f94.png)

What do you think might be the reason behind this?